torch/csrc/jit/mobile/upgrader_mobile.cpp

```diff
@@ -270,3 +317,4 @@ const std::vector<ByteCodeFunctionWithOperator>& getUpgraderBytecodeList() {
 
```

Comment:
where is clang-format-off?

Reply:
`clang-format-off` is added in https://github.com/pytorch/pytorch/pull/71578 and it's merged. I just rebase